addons/mass_mailing/tests/test_mailing_ab_testing.py

```diff
@@ -166,3 +167,11 @@ def test_mailing_ab_testing_minimum_participants(self):
             ab_testing.action_send_mail()
         self.assertEqual(ab_testing.state, 'done')
         self.assertEqual(len(self._mails), 1)
+
+    def test_mailing_ab_testing_duplicate_date(self):
+        """ Test that "Send final on" date value should be copied in new mass_mailing """
+        ab_testing_mail_1 = Form(self.ab_testing_mailing_1)
+        ab_testing_mail_1.ab_testing_schedule_datetime = datetime.now() + timedelta(days=10)
+        action = ab_testing_mail_1.save().action_duplicate()
+        ab_testing_mailing_2 = self.env[action['res_model']].browse(action['res_id'])
+        self.assertEqual(ab_testing_mailing_2.ab_testing_schedule_datetime.strftime('%Y-%m-%d %H:%M:%S'), ab_testing_mail_1.ab_testing_schedule_datetime)
```

Comment:
Better use tools defined in fields.Datetime class, like 'to_datetime' or 'to_string' than doing it manually :)